### ISSUE TITLE
oversized backend: Relation ... cannot be expanded at timestame ...: vector::reserve

### ISSUE THREAD

**Issue by mmd-osm** [drolbr/Overpass-API#557]
We've noticed that the oversized backend on z.overpass-api.de returns an error for the following query, while the same query works ok on lz4.

`<remark> runtime error: Relation 2632934 cannot be expanded at timestamp 2019-02-11T09:29:28Z: vector::reserve </remark>`

```
[adiff:"2019-01-01T00:00:00Z"];
rel(2632934);
out geom meta;
```

My oversized db shows exactly the same error for this query, so this should be reproducible.

**Comment by drolbr** [comment]
Simpler query to provoke the bug:

    [date:"2019-02-11T09:29:27Z"];
    rel(2632934);
    out geom meta;

What I know so far from a post-mortem analysis is that this is caused by the absence of version with expiration timestamp 2019-02-11T11:22:29Z. The corresponding object is basically present but has the wrong timestamp 0000-00-00T00:02:09Z. The object has been written during the catch-up updates, thus all databases that are based on that clone are affected. How this happened is essentially unknown so far.

A sanity check suggests that no other relations have been affected.

**Comment by mmd-osm** [comment]
> thus all databases that are based on that clone are affected

Just for clarification, my database isn’t based on a clone, it’s been created from a 2012 planet and subsequent daily diff updates.

**Comment by drolbr** [comment]
Fixed in 429967a9086a1a5533bebd1b4edeff0e1a376739 and ae21f002ddcba33b5322e80784bb151413630fcd

**Comment by mmd-osm** [comment]
Retest using full db rebuild looks good!
